app/controllers/flyover_comments/comments_controller.rb

```diff
@@ -24,11 +24,10 @@ def index
     def create
       @comment = FlyoverComments::Comment.new(comment_params)
       prepare_for_comment_creation
-      @comment._user = _flyover_comments_current_user
+      @comment.commenter = current_membership || current_user
```

Comment:
This is going to blow up for anything using this gem without the concept of current_membership. I think we should stick to using `_flyover_comments_current_user` locally and override it in Forecasts if needed